api/v3/Mailing.php

```diff
@@ -200,7 +200,10 @@ function civicrm_api3_mailing_clone($params) {
   while ($dao->fetch()) {
     // CRM-11431; account for multi-lingual
     $entity = (substr($dao->entity_table, 0, 15) == 'civicrm_mailing') ? 'mailings' : 'groups';
-    $newParams[$entity][strtolower($dao->group_type)][] = $dao->entity_id;
+    $group_type = strtolower($dao->group_type);
+    if (!in_array($dao->entity_id, $newParams[$entity][$group_type])) {
```

Comment:
`in_array()` without it's third parameter being `TRUE` can produce weird results so preference is always to specify it.

Reply:
Ok @aydun ! I don't know why I never received an email with your comment - or maybe it just got lost in the shuffle. I fixed this now in the latest.